pennylane/ops/qutrit/observables.py

```diff
@@ -133,3 +146,136 @@ def compute_diagonalizing_gates(eigenvectors, wires):  # pylint: disable=argumen
+        if not isinstance(index, int) or index < 1 or index > 8:
+            raise ValueError("The index must be an integer between 1 and 8 inclusive.")
+
+        super().__init__(index, wires, do_queue=do_queue, id=id)
+
+    @staticmethod
+    def compute_matrix(index):  # pylint: disable=arguments-differ
+        r"""Representation of the operator as a canonical matrix in the computational basis (static method).
+
+        The canonical matrix is the textbook matrix representation that does not consider wires.
+        Implicitly, this assumes that the wires of the operator correspond to the global wire order.
+
+        .. seealso:: :meth:`~.GellMannObs.matrix`
+
+        Args:
+            index (int): The index of the Gell-Mann matrix to be used. Must be between 1
+            and 8 inclusive
+
+        Returns:
+            tensor_like: canonical matrix
+
+        **Example**
+
+        >>> qml.GellMannObs.compute_matrix(8)
+        array([[ 0.57735027+0.j,  0.        +0.j,  0.        +0.j],
+               [ 0.        +0.j,  0.57735027+0.j,  0.        +0.j],
+               [ 0.        +0.j,  0.        +0.j, -1.15470054+0.j]])
+        """
+        return gm_mats[index - 1]
+
+    @staticmethod
+    def compute_eigvals(index):  # pylint: disable=arguments-differ
+        r"""Eigenvalues of the operator in the computational basis (static method).
+
+        If :attr:`diagonalizing_gates` are specified and implement a unitary :math:`U`,
+        the operator can be reconstructed as
+
+        .. math:: O = U \Sigma U^{\dagger},
+
+        where :math:`\Sigma` is the diagonal matrix containing the eigenvalues.
+
+        Otherwise, no particular order for the eigenvalues is guaranteed.
+
+        .. seealso:: :meth:`~.GellMannObs.eigvals`
+
+        Args:
+            index (int): The index of the Gell-Mann matrix to be used. Must be between 1
+            and 8 inclusive
+
+        Returns:
+            array: eigenvalues
+
+        **Example**
+
+        >>> qml.GellMannObs.compute_eigvals(1)
+        [1. -1.  0.]
+        """
+        if index != 8:
+            return np.array([1, -1, 0])
+
+        return np.array([1, 1, -2]) / np.sqrt(3)
+
+    @staticmethod
+    def compute_diagonalizing_gates(
+        index, wires
+    ):  # pylint: disable=arguments-differ,unused-argument
+        r"""Sequence of gates that diagonalize the operator in the computational basis (static method).
+
+        Given the eigendecomposition :math:`O = U \Sigma U^{\dagger}` where
+        :math:`\Sigma` is a diagonal matrix containing the eigenvalues,
+        the sequence of diagonalizing gates implements the unitary :math:`U`.
+
+        The diagonalizing gates rotate the state into the eigenbasis
+        of the operator.
+
+        .. seealso:: :meth:`~.GellMannObs.diagonalizing_gates`.
+
+        Args:
+            index (int): The index of the Gell-Mann matrix to be used. Must be between 1
+            and 8 inclusive
```

Comment:
```suggestion
            index (int): The index of the Gell-Mann matrix to be used. Must be between 1 and 8 inclusive
```
Applies to all instances of this docstring throughout the file. Either indent the wrapped part four spaces so it doesn't align with the names of the arguments, or just make this one line, since it's really not that long.